proposed/http-client/http-client.md

```diff
@@ -15,6 +15,10 @@ The goal of this PSR is to allow developers to create libraries decoupled from H
 implementations. This would make libraries more stable since the reduced number of 
 dependencies and the likelihood to get in version conflicts is reduced.
 
+The second goal is that all HTTP clients should follow [Liskov substitutions principle][Liskov]. 
```

Comment:
"...should follow the Listov substitution..." - i think a **the** is missing here